generator/config/search/geoShape.yaml

```diff
@@ -0,0 +1,171 @@
+# $schema: ../schema.json
+name: geoShape
+link: 'https://www.mongodb.com/docs/atlas/atlas-search/geoShape/'
+type:
+    - searchOperator
+encode: object
+description: |
+    The geoShape operator supports querying shapes with a relation to a given 
+    geometry if indexShapes is set to true in the index definition.
+arguments:
+    -
+        name: path
+        type:
+            - searchPath
+    -
+        name: relation
+        type:
+            - string # contains | disjoint | intersects | within
```

Comment:
Curious why you enumerated the possible values here, but not for the tokenOrder string earlier.

Reply:
Updating tokenOrder